README.md

```diff
@@ -0,0 +1,11 @@
+# kyuubi
```

Comment:
Capital K

Reply:
Maybe a good name should be Kyuubi-k8s-operator. What do you think @deusebio ?

Reply:
Taking reference from the title in history server repo, shall I rename to "Charmed Kyuubi Operator"?

Reply:
Title changed to "Kyuubi K8s Operator" for now.